src/lib/AnnotationControls.ts

```diff
@@ -94,6 +94,14 @@ export default class AnnotationControls {
      */
     private handleFullscreenExit = (): void => this.handleFullscreenChange(false);
 
+    /**
+     * Returns the current active annotation mode
+     * @return {string} Current active annotation mode
```

Comment:
The type is `AnnotationMode` right? And I don't think js doc is needed if we have `AnnotationMode` as return type below.

Reply:
Sounds good, I'll remove it then